mypy_django_plugin/transformers/models.py

```diff
@@ -83,6 +85,81 @@ def get_generated_manager_info(self, manager_fullname: str, base_manager_fullnam
         # Not a generated manager
         return None
 
+    def get_or_create_manager_with_any_fallback(self, related_manager: bool = False) -> TypeInfo:
+        """
+        Create a Manager subclass with fallback to Any for unknown attributes
+        and methods. This is used for unresolved managers, where we don't know
+        the actual type of the manager.
+
+        The created class is reused if multiple unknown managers are encountered.
+        """
+
+        name = "UnknownManager" if not related_manager else "UnknownRelatedManager"
+
+        # Check if we've already created a fallback manager class for this
+        # module, and if so reuse that.
+        manager_info = self.lookup_typeinfo(f"{self.model_classdef.info.module_name}.{name}")
+        if manager_info and manager_info.metadata.get("django", {}).get("any_fallback_manager"):
+            return manager_info
+
+        fallback_queryset = self.get_or_create_queryset_with_any_fallback()
+        base_manager_fullname = (
+            fullnames.MANAGER_CLASS_FULLNAME if not related_manager else fullnames.RELATED_MANAGER_CLASS
+        )
+        base_manager_info = self.lookup_typeinfo(base_manager_fullname)
+        assert base_manager_info, f"Type info for {base_manager_fullname} missing"
+
+        base_manager = fill_typevars(base_manager_info)
+        assert isinstance(base_manager, Instance)
+        manager_info = self.add_new_class_for_current_module(name, [base_manager])
+        manager_info.fallback_to_any = True
+
+        # For methods on BaseManager that return a queryset we need to update
+        # the return type to be the actual queryset subclass used. This is done
+        # by adding the methods as attributes with type Any to the manager
+        # class. The actual type of these methods are resolved in
+        # resolve_manager_method.
+        for method_name in MANAGER_METHODS_RETURNING_QUERYSET:
+            helpers.add_new_sym_for_info(manager_info, name=method_name, sym_type=AnyType(TypeOfAny.special_form))
+
+        manager_info.metadata["django"] = django_metadata = {
+            "any_fallback_manager": True,
+            "from_queryset_manager": fallback_queryset.fullname,
+        }
+        if not related_manager:
+            django_metadata["from_queryset_managers"] = {
+                f"django.db.models.manager.{manager_info.fullname}": manager_info.fullname,
+            }
```

Comment:
I _think_ this bit can be removed, as we're not going to try to do a dynamic lookup of this manager?

Reply:
Yes, it's only on the final iteration, and I'm gonna remove this. Wasn't entirely clear to me what this was used for when I wrote it, but it's clear now that this is not needed :)